LICENSE

```diff
@@ -277,6 +277,7 @@ commons-collections:commons-collections
 commons-io:commons-io
 commons-logging:commons-logging
 commons-net:commons-net
+dev.failsafe:failsafe
```

Comment:
Question: Is it a new transitive dependency brought by iceberg 1.7?

Reply:
yup, it's added as part of AWS S3InputStream retry handling via : https://github.com/apache/iceberg/pull/10433/files